src/lib/features/playground/advanced-playground.test.ts

```diff
@@ -0,0 +1,254 @@
+import {
+    IUnleashTest,
+    setupAppWithCustomConfig,
+} from '../../../test/e2e/helpers/test-helper';
+import dbInit, { ITestDb } from '../../../test/e2e/helpers/database-init';
+import getLogger from '../../../test/fixtures/no-logger';
+
+let app: IUnleashTest;
+let db: ITestDb;
+
+beforeAll(async () => {
+    db = await dbInit('advanced_playground', getLogger);
+    app = await setupAppWithCustomConfig(
+        db.stores,
+        {
+            experimental: {
+                flags: {
+                    advancedPlayground: true,
```

Comment:
While we are at it, perhaps strictSchemaValidation?

Reply:
Unless it touches too difficult schemas, like strategies etc, which need more work.

Reply:
we'll be adding openapi in the next PR so flag will be added only then